provider/marathon/marathon_test.go

```diff
@@ -266,6 +384,7 @@ func TestMarathonLoadConfigNonAPIErrors(t *testing.T) {
 
 			c.application.ID = "/app"
 			c.task.ID = "task"
+			c.task.AppID = c.application.ID
```

Comment:
I suppose you can undo this once you've moved the test cases into the correct test.